adagrams/game.py

```diff
@@ -1,11 +1,125 @@
+import random
+
 def draw_letters():
-    pass
+    letters = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+    hand = []
+    for i in range(10):
```

Comment:
So you might get a difference of opinion here depending on who you ask about this for loop, but I think there's a couple different ways to write it that makes your code a little more self-documenting. 

1) you could create a variable LENGTH_OF_HAND = 10, then you could have `for i in range(length_of_hand):`
2) you could create a variable LENGTH_OF_HAND = 10, then you could have a while loop instead `while LENGTH_OF_HAND < 10:`

They all accomplish the same thing, but being strategic about using variables that are descriptively named helps others reading your code understand what's going on more quickly. For instance, if I had no idea how Adagrams was played, I might wonder "why range(10)? what's the significance of 10?"

Reply:
Thanks!